src/software/ai/hl/stp/play/play.cpp

```diff
@@ -164,6 +164,50 @@ std::unique_ptr<TbotsProto::PrimitiveSet> Play::get(
         }
     }
 
+    /* substitute injured robots when the game is in play */
+    if (injured_robots.size() > 0)
+    {
+        std::map<RobotId, TrajectoryPath> robot_trajectories = {};
+        unsigned int num_injured = (unsigned int)injured_robots.size();
+        num_tactics -= num_injured;
+
+        // substitution tactic is just a move tactic to a decided location
+        std::shared_ptr<MoveTactic> auto_sub_tactic = std::make_shared<MoveTactic>();
+
+        // move to middle of court and to positive y boundary and stop
+        auto_sub_tactic->updateControlParams(
+            Point(0, world_ptr->field().totalYLength() / 2), Angle::zero(),
```

Comment:
we need to be able to control which boundary to go to. Ideally, via dynamic params

Reply:
bump @Mr-Anyone